Makefile

```diff
@@ -1,5 +1,10 @@
 .PHONY: docs
 
+install:
+	poetry install --extras full --with docs --no-ansi --sync
+	pre-commit install
+.PHONY: install
```

Comment:
One command install for newcomers. Is the default when running `make` without arguments.